package.json

```diff
@@ -5,7 +5,7 @@
   "leadMaintainer": "Volker Mische <volker.mische@gmail.com>",
   "main": "src/index.js",
   "scripts": {
-    "lint": "aegir lint",
+    "lint": "aegir lint && flow check --color=always",
```

Comment:
I'd like to start slow. So type checking would be its own command `type-check` or so. This way it also won't run on CI yet. I'd like to address @mikeal concerns about new contributors. I'd like to add the type check as separate check so that it's also clear to contributors that their code basically works and that it's just the types.

Reply:
Can we add this to at least "release" script ?

Reply:
We could even add it to CI, I just want it to be a distinct test step.

Reply:
@vmx so I've added  `"check": "flow check --color=always"`. I would like it to run before pull is being submitted, however I'm not exactly sure what the convention here is (I tend to have `npm test` setup to do all the checks) so please advice.

Reply:
Never mind, I've figured it out https://github.com/multiformats/js-cid/pull/82/commits/1bc16700542c4289236bb8508154b2d0abb23d55